src/python_testing/TC_RR_1_1.py

```diff
@@ -442,31 +418,33 @@ async def test_TC_RR_1_1(self):
             await self.fill_user_label_list(dev_ctrl, self.dut_node_id)
         else:
             logging.info("Step 9: Skipped due to no UserLabel cluster instances")
-
-        # Step 10: Count all group cluster instances
+        # Step 10: Reconfig ACL to allow test runner access to Groups clusters on all endpoints.
+        logging.info("Step 10: Reconfiguring ACL to allow access to Groups Clusters")
+        self.send_acl(test_step=10, client_by_name=client_by_name, enable_access_to_group_cluster=False, fabric_table=fabric_table)
```

Comment:
enable_access_to_group_cluster shall be True?

Reply:
Ops... that is correct